tests/subsys/canbus/isotp/conformance/src/main.c

```diff
@@ -296,14 +370,18 @@ static void prepare_cf_frames(struct frame_desired *frames, size_t frames_cnt,
 
 	for (i = 0; i < frames_cnt && remaining_length; i++) {
 		frames[i].data[0] = CF_PCI_BYTE_1 | ((i+1) & 0x0F);
-		frames[i].length = CAN_DL;
+		frames[i].length = TX_DL;
 		memcpy(&des_frames[i].data[1], data_ptr, DATA_SIZE_CF);
```

Comment:
Not a bug from these changes but `&des_frames[i]` here should be `frames`, and the same thing a few lines below this.
It has no impact on the existing tests, but maybe worth tidying up

Reply:
Yeah, you are right. I've added a commit to fix this bug.